src/docbuilder/rustwide_builder.rs

```diff
@@ -678,7 +703,9 @@ impl RustwideBuilder {
     }
 
     fn get_repo(&self, metadata: &MetadataPackage) -> Result<Option<i32>> {
-        self.repository_stats_updater.load_repository(metadata)
+        self.repository_stats_updater
+            .load_repository(metadata)
+            .map_err(Into::into)
```

Comment:
Hmm, why not return a `failure::Result` from `load_repository`?

Reply:
that's probably the problem with the partial migration without rustwide :) 

in the end, I wanted most part migrated, that includes the repository stats :)